src/rustdoc/README.md

```diff
@@ -0,0 +1,20 @@
+# Rustdoc
+Rust's rustdoc team members are responsible for maintaining the Rustdoc tool, improving its performance
+and considering the stabilization of rustdoc features.
+
+We use the Forge to document the team's processes, policies and working practices, if you'd like to
+read about how the compiler and rustdoc work and instructions on how to set up a development environment,
+you're looking for the [rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/).
+
+- [Calendar](./calendar.md)
+  - *How do I subscribe to the compiler team's calendar?*
```

Comment:
Thanks for applying https://github.com/rust-lang/rust-forge/pull/852#discussion_r2100561697 and mentioning the calendar!

However, does it actually warrant it's own section/file. Why not merge it into the *Meetings* section? Surely most if not all calendar entries will always be about meetings?

Reply:
Let's check that in a follow-up since it implies coherency with other teams.